libr/include/experimental/x_m1.h

```diff
@@ -0,0 +1,26 @@
+#ifndef X_M1_H
+#define X_M1_H
+
+#include <r_bin.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+//R_API void r_bin_sorted_section_free (void  /*RBinSortedSection*/ *data);
+//R_API int r_bin_sorted_section_vaddr_cmp (const RBinSortedSection *a, const RBinSortedSection *b);
+//R_API int r_bin_sorted_section_paddr_cmp (const RBinSortedSection *a, const RBinSortedSection *b);
+//R_API int r_bin_sorted_section_contains_addr (const RBinSortedSection *a, ut64 off, int va);
+R_API ut64 r_bin_section_get_from_addr (RBinObject *o, RBinSection *s, int va);
```

Comment:
why not return an interval? and use the same thing we have for io?

i think the current interval api is a bit dirty, but we must be consistent and clean up.

also better have a single function to get from/to in a single call even if we just want the one.